src/state/nftMarket/helpers.ts

```diff
@@ -0,0 +1,29 @@
+import { request, gql } from 'graphql-request'
+import { GRAPH_API_MFTMARKET } from 'config/constants/endpoints'
+
+export const getCollections = async (): Promise<any[]> => {
+  try {
+    const res = await request(
+      GRAPH_API_MFTMARKET,
+      gql`
+        {
+          collections {
+            id
+            name
+            symbol
+            active
+            totalVolumeBNB
+            numberTokensListed
+            tradingFee
+            creatorFee
+          }
+        }
+      `,
+    )
+
+    return res.collections
+  } catch (error) {
+    console.error(error)
```

Comment:
Just a preference, but I usually like to put string describing where error came from, like
```javascript
console.error('Failed to fetch NFT collections', error)
```

Otherwise it takes a while to figure out where exactly in code this error was raised